source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,202 @@
+=================
+Change a Document
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+You can change documents in a MongoDB collection using **update** and
+**replace** operations. Update operations modify only the fields that
+you specify in one or more documents while leaving other fields and
+values unchanged. Replace operations remove all existing fields in one
+or more documents and substitute the deleted fields with new fields and
+values.
+
+.. _updateDocuments:
+
+Update
+------
+
+To update one or more documents, create an **update document** that
+includes an **update operator**, which specifies the type of update to
+perform and the fields and values that describe the change. Update
+documents use the following format:
+
+.. code-block:: go
+
+   bson.D{{"<update operator>", bson.D{{"<field>", "<value>"}, {"<field>", "<value>"}, ... }}, {"<update operator>", ... }, ... }
+   
+
+You can use one or more of the following update operators in your update
+document:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Update Operator
+     - Description
+
+   * - ``$set``
+     - replaces the value of a field with a specified value or creates a new
+       field with a specified value
+
+   * - ``$inc``
+     - increments or decrements field values
+
+   * - ``$rename``
+     - renames fields
+
+   * - ``$unset``
+     - removes fields
+     
+   * - ``$mul``
+     - multiplies the value of a field by a specified number
```

Comment:
Suggestion: There are 9 update operators, so I think it's a bit awkward to omit the ones not already here. And since the link right under this table has all the operators, a brief description then linking to it should be enough.

Reply:
got it